ansible/configs/cloud-credentials/post_infra.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Step 002 Post Infrastructure
+  hosts: localhost
+  connection: local
+  become: false
+  gather_facts: false
+  tags:
+    - step002
+    - post_infrastructure
+  tasks:
+    - debug:
+      msg: "Step 000 Pre Infrastructure"
+
+    - name: Add AWS sandbox credentials if enabled
+      when:
+      - cloud_provider == 'ec2'
+      block:
+      - name: Set facts for AWS credentials
+        set_fact:
+          student_access_key_id: "{{ cloudformation_out_final.stack_outputs.StudentUserAccessKey }}"
+          student_secret_access_key: "{{ cloudformation_out_final.stack_outputs.StudentUserSecretAccessKey }}"
+      
+      - name: Add AWS credentials with agnosticd_user_info
+        agnosticd_user_info:
+          msg: "{{ item }}"
+          data:
+            aws_access_key_id: "{{ student_access_key_id }}"
+            aws_secret_access_key: "{{ student_secret_access_key }}"
+            aws_subdomain_base_suffix: "{{ subdomain_base_suffix }}"
+          loop:
+            - "aws_access_key_id: {{ student_access_key_id }}"
+            - "aws_secret_access_key: {{ student_secret_access_key }}"
+            - "aws_sandbox_zone: {{ subdomain_base_suffix }}"
```

Comment:
Setting `msg` and `data` should be separate tasks. Setting `data` should not have a loop.